easybuild/easyblocks/i/impi.py

```diff
@@ -141,6 +143,22 @@ def post_install_step(self):
             for script in [os.path.join(script_path, 'mpivars.sh') for script_path in script_paths]:
                 apply_regex_substitutions(os.path.join(self.installdir, script), regex_subs)
 
+        # Recompile libfabric if necessary
+        if impiver >= LooseVersion('2019.0.0') and self.cfg['rebuild_libfabric']:
+            if self.cfg['ofi_internal']:
+                os.chdir(os.path.join(self.installdir, 'libfabric'))
+                extract_file('src.tgz', './')
+                libfabric_installpath = os.path.join(self.installdir, 'intel64', 'libfabric')
+                cmds = [
```

Comment:
I don't particularly like recreating `ConfigureMake` here, but it might be better than importing another easyblock. That looks even more awkward.

Reply:
Yeah, it may be difficult to leverage `ConfigureMake` here, since you'll need to convince it to *not* pick up on `configopts`, `buildopts`, etc. Also, lot's of the logic in there is totally irrelevant here.

This isn't too terrible (and chances are that the build procedure for `libfabric` will change anyway)...